dbt_project.yml

```diff
@@ -22,6 +21,8 @@ vars:
     component: "{{ ref('stg_jira__component') }}"
     field: "{{ ref('stg_jira__field') }}"
     sprint: "{{ ref('stg_jira__sprint') }}"
+    include_comments: true  # this package aggregates issue comments so that you have a single view of all your comments in the jira__issue_enhanced table. This can cause limit errors if you have a large dataset. Disable to remove this functionality.
+    using_sprints: true # disable if you are not using sprints in Jira
```

Comment:
same comment as in the source package -- since we're using `true` as the default value when we call these variables, we don't necessarily need to have them here in the dbt_project.yml

Reply:
As discussed in stand-up, we'll keep these here we'll explicitly define any variables we create in the dbt_project.yml file to increase clarity.